src/Container/Container.d.ts

```diff
@@ -1,18 +1,22 @@
-import { SvelteComponent } from 'svelte';
+declare module 'sveltestrap' {
+  import { SvelteComponent } from 'svelte';
+  import { HTMLDivAttributes } from 'svelte/elements';
```

Comment:
I don't think `svelte/elements` exports directly, does it?

I've been using:
```ts
import { HTMLAttributes } from 'svelte/elements';

export interface ContainerProps extends HTMLAttributes<HTMLDivElement> {
```

Reply:
honestly no clue. followed a pattern in. `Button.svelte`. 

`import { HTMLButtonAttributes } from 'svelte/elements';`
`export interface ButtonProps extends HTMLButtonAttributes {`

switching it over now

Reply:
<img src="https://media2.giphy.com/media/vX9WcCiWwUF7G/giphy.gif" />

for blindly copying and pasting